lib/gui/app.js

```diff
@@ -210,9 +210,16 @@ app.run(($timeout) => {
     // otherwise we risk presenting the same error over
     // and over again to the user, while also heavily
     // spamming our error reporting service.
-    driveScanner.stop()
 
-    return exceptionReporter.report(error)
+    // Clear the drive list, to avoid displaying stale data
+    $timeout(() => {
+      availableDrives.setDrives([])
+    })
+
+    // Only send the first error, and continue scanning
+    if (!driveScanner.errorCount) {
+      analytics.logException(error)
+    }
```

Comment:
I wonder if it'd be worth doing something a bit cleverer and only avoid sending _duplicate_ errors, rather than avoiding sending every error after the first?